src/core/io/tests/4C_io_input_spec_test.cpp

```diff
@@ -705,9 +705,9 @@ description: 'one_of {a, b, one_of {c, one_of {d, e, f}}}'
             {.default_value = std::vector<std::optional<double>>{1., std::nullopt, 3.}, .size = 3}),
         one_of({
             all_of({
-                parameter<std::map<std::string, std::string>>(
-                    "b", {.default_value = std::map<std::string, std::string>{{"key", "abc"}},
-                             .size = 1}),
+                parameter<std::map<std::string, std::string>>("string to string",
+                    {.default_value = std::map<std::string, std::string>{{"key", "abc"}},
```

Comment:
And I also found this bug in the tests: Of course we cannot have another "b" here.